lib/nexmo_developer/config/routes.rb

```diff
@@ -102,6 +102,13 @@
 
   get '/ed', to: 'static#blog_cookie'
 
+  if LoadConfig.exist?('config/engines.yml')
```

Comment:
This is currently at the end of the routes file. Should engines take precedence over existing routes?

Reply:
That's up to us, routes are matched in the order they are specified. So if we want them to take precedence we should move them up.